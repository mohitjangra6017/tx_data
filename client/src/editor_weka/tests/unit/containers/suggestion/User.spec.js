/**
 * This file is part of Totara Enterprise Extensions.
 *
 * Copyright (C) 2020 onwards Totara Learning Solutions LTD
 *
 * Totara Enterprise Extensions is provided only to Totara
 * Learning Solutions LTD's customers and partners, pursuant to
 * the terms and conditions of a separate agreement with Totara
 * Learning Solutions LTD or its affiliate.
 *
 * If you do not have an agreement with Totara Learning Solutions
 * LTD, you may not access, use, modify, or distribute this software.
 * Please contact [licensing@totaralearning.com] for more information.
 *
 * @author Kian Nguyen <kian.nguyen@totaralearning.com>
 * @module editor_weka
 */

import User from 'editor_weka/components/suggestion/User';
import { mount } from '@vue/test-utils';

describe('editor_weka/components/suggestion/User.vue', function() {
  let wrapper = null;

  beforeAll(function() {
    wrapper = mount(User, {
      mocks: {
        $str(id, component) {
          return `${id}, ${component}`;
        },

        $id(random) {
          return `something-${random}`;
        },

        $apollo: {
          loading: false,
        },
      },

      propsData: {
        x: 1,
        y: 1,
        pattern: 'bolobala',
      },
    });
  });

  it('Checks snapshot', function() {
    expect(wrapper.element).toMatchSnapshot();
  });
});
