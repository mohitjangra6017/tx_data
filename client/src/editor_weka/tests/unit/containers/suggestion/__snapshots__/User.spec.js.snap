// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`editor_weka/components/suggestion/User.vue Checks snapshot 1`] = `
<div
  class="tui-editorWeka-user"
  style="left: 1px; top: 1px;"
>
  <div
    class="tui-dropdown tui-dropdown--open"
  >
    <!---->
     
    <div
      aria-labelledby="something-trigger"
      aria-orientation="vertical"
      class="tui-dropdown__menu"
      role="menu"
    >
      <div
        class="tui-dropdown__content tui-dropdown__content--separator"
      >
        <span
          class="sr-only"
        >
          
      matching_users, editor_weka:
    
        </span>
         
      </div>
    </div>
  </div>
</div>
`;
